appveyor.yml

```diff
@@ -8,15 +8,9 @@ environment:
     - PYTHON: "C:\\Python36"
       TOXENV: "docs"
 
-    - PYTHON: "C:\\Python27"
-      TOXENV: "py27"
-
     - PYTHON: "C:\\Python36"
       TOXENV: "py36"
 
-    - PYTHON: "C:\\Python37"
```

Comment:
Any specific reason to do not test against `python 3.7`?

Reply:
No